modules/user/user.controllers.js

```diff
@@ -23,6 +28,29 @@ const mixins = {
     }
     return data;
   },
+
+  async loginUsingWallet(signature, signPayload, { clientIpAddress }) {
+    const { address } = WalletUtils.validateSignature(signature, signPayload, {
+      ip: clientIpAddress,
+      secret,
+    });
+    let user = await this.table.findOne({
+      where: {
+        wallet_address: address,
+      },
+    });
+    let data = await this.loginSuccess(user.id, clientIpAddress);
```

Comment:
what does loginSuccess() do?

Reply:
login success is called to get access-token after verifying signature

Reply:
@abiskar-sapkota